ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoSpec.scala

```diff
@@ -436,7 +436,8 @@ class JdbcLedgerDaoSpec
         firstUploadResult shouldBe PersistenceResponse.Ok
         secondUploadResult shouldBe PersistenceResponse.Ok
         loadedPackages.values.flatMap(_.sourceDescription.toList) should contain theSameElementsAs
-          Seq(firstDescription) ++ Seq.fill(17)(secondDescription)
+          Seq(firstDescription) ++ Seq.fill(JdbcLedgerDaoSpec.Fixtures.packages.length - 1)(
+            secondDescription)
```

Comment:
This might be easier to read if it looked something like this:

```scala
val actualDescriptions = loadedPackages.values.flatMap(_.sourceDescription.toList)
actualDescriptions should have length JdbcLedgerDaoSpec.Fixtures.packages.length
val (actualFirstDescription :: actualOtherDescriptions) = actualDescriptions
actualFirstDescription should be(firstDescription)
all(actualOtherDescriptions) should be(secondDescription)
```

Reply:
Thanks, changed!

Reply:
This doesn’t work since the order isn’t fixed so I undid this change again.

Reply:
Boooo. Fair enough.